autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
Why is it added?  Throwing in arithmetic complicates (slightly) a simple concept.

Shouldn't `&verbose` simply override it temporarily the same way `:verbose` does?  IMO, `:verb Neomake` is the same as `let g:neomake_verbose = 1 | Neomake | unlet! g:neomake_verbose`.